cmd/ctl/pkg/check/api/api.go

```diff
@@ -92,41 +89,40 @@ func NewCmdCheckApi(ctx context.Context, ioStreams genericclioptions.IOStreams)
 		Use:   "api",
 		Short: "Check if the cert-manager API is ready",
 		Long:  checkApiDesc,
-		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := o.Complete(); err != nil {
-				return err
-			}
-			o.Run(ctx)
-			return nil
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(o.Complete())
+			cmdutil.CheckErr(o.Run(ctx))
```

Comment:
These `cmdutil.CheckErr` functions are also used in most of our other sub-commands.